### PR TITLE
Player input add drag

### PR DESCRIPTION
**Pull request by bambong**
refactoring Select Drag ,Drag  into PlayerInput , and Add Attack Card Effect

### CODE REVIEW

File: Assets/Scripts/PlayerInput.cs [hunk 1]
```diff
@@ -40,28 +43,104 @@ void PlayerInputCheck()
...
+                if (hit.collider != null)
+                {
+
+                    hit.collider.GetComponent<SpriteRenderer>().color = Color.blue;
+
+                }
+
+            }
+
+            selectCard.GetRect().localPosition = selectCardOriginPos;
```
Comment (nick-choi) [inline Assets/Scripts/PlayerInput.cs:137]: These lines are repeated. I recommend extract them to one method. 
```
private void SetCardTransform(Card card, Vector3 position, Vector3 scale)
{
    card.GetRect().localPosition = position;
    card.GetRect().localScale = scale;
}
```

File: Assets/Scripts/PlayerInput.cs [hunk 2]
```diff
@@ -40,28 +43,104 @@ void PlayerInputCheck()
             {
                 
                 selectCard = grResults[0].gameObject.GetComponent<Card>();
-                selectCardOriginPos = selectCard.GetRect().anchoredPosition;
-                selectCard.GetRect().anchoredPosition = Input.mousePosition;
+                selectCardOriginPos = selectCard.GetRect().localPosition;
+                selectCard.GetRect().localScale = new Vector3(1.5f, 1.5f, 1);
```
Comment (nick-choi) [inline Assets/Scripts/PlayerInput.cs:47]: Define constant and use it like below. Don't make a magic number. 
```
private static readonly Vector3 SELECT_CARD_SCALE = new Vector3(1.5f, 1.5f, 1);
```

**Comment by nick-choi** [comment]
plz check coding style.

**Comment by sarahk1280** [comment]
I merged this branch and fixed all conflicts with master

File: Assets/Scripts/PlayerInput.cs (hunk 2 above)
Comment (bambong) [reply Assets/Scripts/PlayerInput.cs]: https://github.com/SuperRookies/Slay24/pull/13/commits/868a101c73371ce4296223fbbb434f557b2d03d2

File: Assets/Scripts/PlayerInput.cs (hunk 1 above)
Comment (bambong) [reply Assets/Scripts/PlayerInput.cs]: https://github.com/SuperRookies/Slay24/pull/13/commits/868a101c73371ce4296223fbbb434f557b2d03d2

**Comment by nick-choi** [comment]
LGTM

**Pull request merged**